packages/e2e/README.md

```diff
@@ -69,7 +71,7 @@ cd example
 flutter drive --driver=test_driver/<package_name>_test.dart test/<package_name>_e2e.dart
 ```
 
-You can run tests on web on release mode.
+You can run tests on web on release or profile mode.
```

Comment:
nit: I think we generally say "in release mode" rather than "on release mode"

Reply:
done.